util/fibers/fiber2.h

```diff
@@ -109,6 +109,14 @@ uint64_t FiberLongRunSumUsec() noexcept;
 // It is advised to call this function when a program starts.
 void SetDefaultStackResource(PMR_NS::memory_resource* mr, size_t default_size = 64 * 1024);
 
+// Returns the total stack size (virtual memory) for worker fibers for the current thread.
+// Please note that RSS memory usage is usually smaller, depending on the actuall stack usage
+// of the fibers.
+size_t WorkerStackSize();
```

Comment:
I'd rename this to `WorkerFibersStackSize()` (I proposed that in the previous iteration but perhaps you've missed it because I added another proposal :)